internal/cli/server/config.go

```diff
@@ -996,8 +996,7 @@ func (c *Config) buildNode() (*node.Config, error) {
 	}
 
 	if c.P2P.NoDiscover {
-		// Disable networking, for now, we will not even allow incomming connections
-		cfg.P2P.MaxPeers = 0
+		// Disable peer discovery
 		cfg.P2P.NoDiscovery = true
```

Comment:
What is the difference in outcome in MaxPeers=0 and NoDiscovery=true?

Reply:
max peers 0 will not let anyone to connect with it and it wont connect to everyone
no discovery will not let anyone recognize the node but the node can connect to others
good for validators as we don't want any unknown nodes to connect to it even if somehow they get our node details